src/starcheck.pl

```diff
@@ -136,6 +138,23 @@
 my $dither_file= get_file("$par{dir}/History/DITHER.txt*",'dither'); 
 my $radmon_file= get_file("$par{dir}/History/RADMON.txt*", 'radmon');
 
+# Check for characteristics.  Ignore the get_file required vs not API and just pre-check
+# to see if there is characteristics
+my $char_file;
+my $char_glob = "$par{dir}/${sosa_dir_slash}mps/ode/characteristics/CHARACTERIS*";
```

Comment:
Brent indicated that this was not going to be in `vehicle/` so I don't think the sosa dir bit should be there.